content/kms-plugins.md

```diff
@@ -0,0 +1,39 @@
++++
+title = "KMS Plugins for Sigstore"
+author = "Ramon Petgrave (@ramonpetgrave64)"
+date = "2025-02-19"
```

Comment:
can we update this to indicate 4-4-2025 and we'll publish this tomorrow?

Reply:
Changed to Monday Apr 7, in case we want to mention another provider.
 - https://github.com/sigstore/sigstore-blog/pull/72/files#r2027570830